src/openms/include/OpenMS/ANALYSIS/ID/IDConflictResolverAlgorithm.h

```diff
@@ -45,7 +45,7 @@
 #include <algorithm>
 
 using namespace OpenMS;
```

Comment:
actually, this is just as bad - if someone imports this, all `OpenMS::` namespace is exposed ...

Reply:
Agreed. However, this simple pull request was to understand more about the project maintainers' view on this. 
Judging from your PR: https://github.com/OpenMS/OpenMS/pull/3732, it seemed that these clean-ups do not "rock the boat" (in terms of not wasting the project's maintainers' time).
Thanks for the feedback and encouragement!